assets/styl/_modules/_icon.styl

```diff
@@ -43,9 +43,8 @@
 
     for key, value in $colors
         &-{key}
-            icon-text-color = _contrast-check($color-white, value) ? $color-white : $color-black
             background-color value
-            color icon-text-color
+            color _contrast_choose(value, $color-white $color-black)
```

Comment:
More I think about if, less I think to have a default value for $list is a good idea